guides/doc-Planning_for_Project/topics/Introduction.adoc

```diff
@@ -217,7 +217,7 @@ ifndef::satellite[]
 [[Foreman_Server_Operating_System]]
 ==== {ProjectServer} Operating System
 
-{Project} has packages for CentOS 7 & 8 and clones, and Debian and clones.
+{Project} has packages for CentOS 8 and clones, and Debian 11 and Ubuntu 20.04.
```

Comment:
Is this centos 8 stream?

Reply:
Well yes, it came from https://github.com/theforeman/foreman-documentation/pull/982. It should probably be Enterprise Linux but I thought this was a good first step.

Reply:
OK, works for me!